.coderabbit.yaml

```
@@ -0,0 +1,240 @@
+    - "*.tsx"
+    - "*.js"
+    - "*.jsx"
+    - "*.css"
+    - "*.scss"
+    - "*.json"
+    - "*.md"
+    - "package.json"
+    - "tsconfig.json"
+    - "tailwind.config.ts"
+    - "vite.config.ts"
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)